tasks/transform_conformance/exec_runner.snap.md

```diff
@@ -0,0 +1,96 @@
+Passed: 350/409
+
+# All Passed:
+* babel-plugin-transform-unicode-sets-regex
```

Comment:
Some of these have no exec.js I assume? Shall we exclude the ones that does not have exec.js?